crates/zkvm/lib/src/utils.rs

```diff
@@ -130,3 +139,57 @@ pub fn bytes_to_words_le(bytes: &[u8]) -> Vec<u32> {
         .map(|chunk| u32::from_le_bytes(chunk.try_into().unwrap()))
         .collect::<Vec<_>>()
 }
+
+/// A trait for affine points on Weierstrass curves.
+pub trait WeierstrassAffinePoint<const N: usize>: AffinePoint<N> {
+    /// The infinity point is set to (0, 0).
+    fn infinity() -> Self {
```

Comment:
This seems kind of bad? it would be good if an affine point is valid.

Reply:
Now represented as a separate type.